lib/jellyfish/config_reader.ex

```diff
@@ -75,34 +76,8 @@ defmodule Jellyfish.ConfigReader do
         _other ->
           fallback.(value)
       end
-    end
-  end
-
-  def read_dist_config() do
-    if read_boolean("JF_DIST_ENABLED") do
-      node_name_value = System.get_env("JF_DIST_NODE_NAME")
-      cookie_value = System.get_env("JF_DIST_COOKIE", "jellyfish_cookie")
-      nodes_value = System.get_env("JF_DIST_NODES", "")
-
-      unless node_name_value do
-        raise "JF_DIST_ENABLED has been set but JF_DIST_NODE_NAME remains unset."
-      end
-
-      node_name = parse_node_name(node_name_value)
-      cookie = parse_cookie(cookie_value)
-      nodes = parse_nodes(nodes_value)
-
-      if nodes == [] do
-        Logger.warning("""
-        JF_DIST_ENABLED has been set but JF_DIST_NODES remains unset.
-        Make sure that at least one of your Jellyfish instances
-        has JF_DIST_NODES set.
-        """)
-      end
-
-      [enabled: true, node_name: node_name, cookie: cookie, nodes: nodes]
     else
-      [enabled: false, node_name: nil, cookie: nil, nodes: []]
+      nil
```

Comment:
We don't need this `else` and because other functions don't add it let's be consistent with them and remove it